pkg/agent/openflow/pipeline.go

```diff
@@ -1153,7 +1154,7 @@ func (f *featureNetworkPolicy) flowsToTrace(dataplaneTag uint8,
 				klog.ErrorS(err, "Failed to get OpenFlow table by tableID", "id", ctx.dropFlow.TableId)
 				continue
 			}
-			dropFlow := f.defaultDropFlow(table, ctx.matchPairs, ctx.dropFlowEnableLogging)
+			dropFlow := f.defaultDropFlow(table, ctx.matchPairs, false)
```

Comment:
why it's hardcoded to false?

Reply:
It is used for trace flow, we do not need to regenrate flows with logging actions for a traceFlow